common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json

```diff
@@ -7,7 +7,12 @@
             "tags":     "co",
             "group":    "overlays",
             "product":  "AIRX2RET_DAY",
-
+            "layergroup": [
+              "airs"
+            ],
```

Comment:
looks like a duplicate

Reply:
duplicate of what?

Reply:
I see it. `layergroup`, I had it happen once or twice when I was doing the auto-find and replace.

Reply:
OK - I've never used this review thing so I missed that :)